scripts/blog.js

```diff
@@ -7,9 +7,18 @@ blog.makePosts = function(){
     blog.sortArticles();
     var post = blog.rawData[i];
     post.daysBetween = blog.dateDiff(post.publishedOn);
-    var posted =new Article(post);
-    posted.toHtml();
+    post.authorSlug = blog.slugify(post.author);
+    new Article(post);
   }
+  blog.toHtml();
+};
+
+blog.slugify = function (string) {
+  return string.replace(/\s/g , '-');
+};
```

Comment:
nice, this can actually be a utility function (util.slugify) since it's not exclusive to blog strings